build.rs

```diff
@@ -1,73 +1,36 @@
-use std::env;
-use std::ffi::{OsStr, OsString};
 use std::path::PathBuf;
 use std::process::Command;
-
-fn find_make() -> OsString {
-    if let Some(make) = env::var_os("MAKE") {
-        return make
-    }
-
-    match Command::new("gmake").status() {
-        Ok(_) => OsStr::new("gmake").to_os_string(),
-        Err(_) => OsStr::new("make").to_os_string(),
-    }
-}
+use std::{env, fs};
 
 fn main() {
     let src = PathBuf::from(env::var_os("CARGO_MANIFEST_DIR").unwrap());
-    let dst = PathBuf::from(env::var_os("OUT_DIR").unwrap());
-
-    // Prevent aclocal being run due to timestamps being messed up by git.
-    // https://stackoverflow.com/questions/18769770/user-of-autotools-generated-tarball-gets-error-message-aclocal-1-13-command-no
-    run(Command::new("touch")
-                .current_dir(src.join("mesa-src"))
-                .arg("configure.ac")
-                .arg("aclocal.m4")
-                .arg("configure")
-                .arg("Makefile.am")
-                .arg("Makefile.in")
-                .arg("src/compiler/glsl/glcpp/glcpp-lex.c")
-                .arg("src/mesa/program/lex.yy.c")
-                .arg("src/compiler/glsl/glcpp/glcpp-parse.c")
-                .arg("src/compiler/glsl/glsl_parser.cpp")
-                .arg("src/mesa/program/program_parse.tab.c")
-                .arg("src/compiler/glsl/glsl_lexer.cpp")
-                .arg("src/glx/indirect.h")
-                .arg("src/glx/indirect_init.c")
-                .arg("src/mapi/glapi/glapi_mapi_tmp.h"));
+    let _dst = PathBuf::from(env::var_os("OUT_DIR").unwrap());
+    let build_folder = src.join("mesa-src").join("build");
```

Comment:
A bit weird to do a build in the srcdir, but fine. Can it be in the out dir instead?